### DISCUSSION TITLE
How to add RetryPolicy for Asp.Net Core app using Linq2Db context

### DISCUSSION THREAD

**Discussion by CatalinCernea** [linq2db/linq2db#4140] [Q&A] [answered]
I saw that there is a possibility to add retry policy for queries.
How can it be done in Asp.Net Core applications?
Should it be added when configuring the db context or in other places?

**Comment by MaceWindu** [comment, marked as answer]
We recommend to add it to context configuration using 
```cs
options.UseRetryPolicy(policyImplementation)
```

see all configuration methods here: https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/Extensions/DataOptionsExtensions.cs#L950-L1090

but you can also set it application-wide:
```cs
Configuration.RetryPolicy.Factory = ...
```
and other Configuration.RetryPolicy.* properties https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/Common/Configuration.cs#L470-L565

**Reply by CatalinCernea** [reply]
Ok. Thanks for clarification.
